prompts/task_read_project_tasks.md

```
@@ -0,0 +1,9 @@
+## Task: Read Project Tasks
+
+### Referenced SOPs:
+*   [SOP: Read Project Tasks](docs/sop_read_project_tasks.md)
+
+### Referenced CRQs:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)